solidity/deploy/41_deploy_multi_asset_vault.ts

```diff
@@ -0,0 +1,46 @@
+import type { DeployFunction } from "hardhat-deploy/types"
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import { waitForTransaction } from "../helpers/deployment"
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { deployments, helpers } = hre
+  const { deployer, governance } = await helpers.signers.getNamedSigners()
+
+  const mezoPortal = await deployments.get("MezoPortal")
+
+  const initialSupportedAssets = [
+    "0x7A56E1C57C7475CCf742a1832B028F0456652F97", // SolvBTC
+    "0xd9D920AA40f578ab794426F5C90F6C731D159DEf", // SolvBTC.BBN
+  ]
```

Comment:
We don't expect to add more tokens, and we don't need to handle testnet tokens or mock them, as we're testing on the main net fork and don't expect to plug this contract into the testnet dApp (at least for now).
Thank you for sharing your experience! I'll keep it in mind in case we want to do something more with this contract.